msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs

```diff
@@ -42,6 +42,12 @@ public class UsernamePasswordIntegrationTests
         public const string Authority = "https://login.microsoftonline.com/organizations/";
         public string[] Scopes = { "User.Read" };
         AuthHelper authHelper = new AuthHelper();
+        
+        [TestInitialize]
+        public void TestInitialize()
```

Comment:
>TestInitialize [](start = 20, length = 14)

It seems to me that you want to run this code once for all tests in this assembly? In that case, note that there are 2 other init attributes in mstest - ClassInitialize and AssemblyInitialize. The last seems more appropriat for your needs: [https://stackoverflow.com/questions/2382552/is-it-possible-to-execute-code-once-before-all-tests-run](https://stackoverflow.com/questions/2382552/is-it-possible-to-execute-code-once-before-all-tests-run)

Reply:
very cool. You're right, i only need this once for the class, so I changed to classInitialize. Thanks for the suggestion. 

---
In reply to: [221246985](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1281#discussion_r221246985) [](ancestors = 221246985)